packages/block-library/src/cover/test/edit.native.js

```diff
@@ -541,7 +541,7 @@ describe( 'color settings', () => {
 		expect( getEditorHtml() ).toMatchSnapshot();
 	} );
 
-	it( 'clears the selected overlay color and mantains the inner blocks', async () => {
+	it( 'clears the selected overlay color and maintains the inner blocks', async () => {
```

Comment:
We'll need to fix this manually:

https://github.com/WordPress/gutenberg/blob/961000076e74a6eea419d79ff8e16ae54a73463b/packages/block-library/src/cover/test/__snapshots__/edit.native.js.snap#L3